packages/contracts/contracts/plugin-trait.clar

```diff
@@ -0,0 +1,6 @@
+(define-trait plugin-trait 
+  (
+    (get-name () (response (string-ascii 50) uint))
+    (get-version () (response uint uint))
```

Comment:
```suggestion
    (get-name? () (response (string-ascii 50) uint))
    (get-version? () (response uint uint))
```
Based on my current understanding, functions that might return an `err` should have a trailing `?` ?

Reply:
This part of the syntax inconsistent in native functions therefore I don't use `?` and `!` in function names.
Example: 
https://docs.stacks.co/references/language-functions#map-get 
https://docs.stacks.co/references/language-functions#element-at

Reply:
Interesting, thanks, I agree.